src/components/shared/PayoutModsList.tsx

```diff
@@ -91,6 +96,12 @@ export default function PayoutModsList({
 
   const hasEditPermission = useHasPermission(OperatorPermission.SetPayoutMods)
 
+  const total_percentage =
```

Comment:
**question (non-blocking)**: we're duplicating this reducer code from `PayModsForm`. Does it make sense to create a shared helper function somewhere and use it here and in `PayModsForm`?